openstack/networking/v2/extensions/vpnaas/services/requests.go

```diff
@@ -0,0 +1,41 @@
+package services
+
+import "github.com/gophercloud/gophercloud"
+
+// CreateOptsBuilder allows extensions to add additional parameters to the
+// Create request.
+type CreateOptsBuilder interface {
+	ToVPNServiceCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts contains all the values needed to create a new VPN service
+type CreateOpts struct {
+	// TenantID specifies a tenant to own the VPN service. The caller must have
+	// an admin role in order to set this. Otherwise, this field is left unset
+	// and the caller will be the owner.
+	TenantID     string `json:"tenant_id,omitempty"`
+	SubnetID     string `json:"subnet_id,omitempty"`
+	RouterID     string `json:"router_id" required:"true"`
+	Description  string `json:"description,omitempty"`
+	AdminStateUp *bool  `json:"admin_state_up"`
+	ProjectID    string `json:"project_id"`
```

Comment:
I don't see `project_id` defined anywhere [here](https://github.com/openstack/neutron-vpnaas/blob/058469e1b99b647537a5228c6a384d93df5484df/neutron_vpnaas/db/vpn/vpn_db.py#L473), so maybe it's an internal neutron field? What happens if `TenantID` and `ProjectID` are different UUIDs?